src/components/StoryFooter/Buttons.js

```diff
@@ -85,8 +85,7 @@ export default class Buttons extends React.Component {
   handleCommentClick = () => {
     const form = document.getElementById('commentFormInput');
     if (form) {
-      form.scrollIntoView(true);
-      document.body.scrollTop -= 200;
+      form.scrollIntoView({ behavior: 'instant', block: 'center', inline: 'start' });
```

Comment:
Problem with using `scrollIntoViewOptions` is that it's experimental technology and not yet widely supported:
![image](https://user-images.githubusercontent.com/1968722/32919592-bb4c64b8-cb26-11e7-9a8d-20c3db3bcd3a.png)
We shouldn't be using it as it might break things for many users.